packages/shared/src/modules/quiz/libs/validation-schemas/quiz-submit-answers.validation-schema.ts

```diff
@@ -0,0 +1,41 @@
+import { sanitizeTextInput } from "shared";
+import z from "zod";
+
+import { QuizCategory, QuizValidationMessage } from "../enums/enums.js";
+
+const ZERO_ARRAY_LENGTH = 0;
+
+const QuizAnswersSchema = z.object({
```

Comment:
Use `camelCase` for schema variable names
```suggestion
const quizAnswersSchema = z.object({
```